cfgov/v1/models/images.py

```diff
@@ -103,6 +103,27 @@ class CFGOVRendition(AbstractRendition):
     def alt(self):
         return self.image.alt
 
+    def get_orientation(self):
+        orientation = 'square'
+        if self.is_portrait:
+            orientation = 'portrait'
+        elif self.is_landscape:
+            orientation = 'landscape'
+
+        return orientation
```

Comment:
I almost think this wants to be a property too —

```python

@property
def orientation(self):
    ...
```
It also might make sense for all of these to be `@cached_property`s too.